<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统配置</string>
  </property>
  <widget class="QGroupBox" name="box_search">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>191</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_search
{
    font-family:Microsoft Yahei;
    color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:5px;
	font: 18pt &quot;.SF NS Text&quot;;
}

QPushButton#btn_search:hover
{
    background-color:rgb(44 , 137 , 255);
}

QPushButton#btn_search:pressed
{
    background-color:rgb(14 , 135 , 228);
    padding-left:3px;
    padding-top:3px;
}</string>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(150, 160, 20);
font: 20pt &quot;.SF NS Text&quot;;
color: rgb(110, 110, 110);</string>
    </property>
    <property name="text">
     <string>条件搜索</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_type_first">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>20</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Adobe Arabic&quot;;
background-color: rgb(166, 166, 166);
border-style:none;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_type_second">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>20</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Adobe Arabic&quot;;
background-color: rgb(166, 166, 166);
border-style:none;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_type_first">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>50</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#label_type_first
{
background-color: rgba(255, 255, 255, 0);
border-style:none;
font: 16pt &quot;.SF NS Text&quot;;
}
QLineEdit#label_type_first:focus
{
	background-color: rgb(255, 255, 127);
}</string>
    </property>
    <property name="text">
     <string>type1</string>
    </property>
   </widget>
   <widget class="QComboBox" name="type_first">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>80</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;.SF NS Text&quot;;</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_type_second">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>120</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#label_type_second
{
background-color: rgba(255, 255, 255, 0);
border-style:none;
font: 16pt &quot;.SF NS Text&quot;;
}
QLineEdit#label_type_second:focus
{
	background-color: rgb(255, 255, 127);
}</string>
    </property>
    <property name="text">
     <string>type2</string>
    </property>
   </widget>
   <widget class="QComboBox" name="type_second">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>150</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;.SF NS Text&quot;;</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <zorder>btn_search</zorder>
   <zorder>label_2</zorder>
   <zorder>box_result</zorder>
   <zorder>box_result</zorder>
   <zorder>btn_type_first</zorder>
   <zorder>btn_type_second</zorder>
   <zorder>label_type_first</zorder>
   <zorder>type_first</zorder>
   <zorder>label_type_second</zorder>
   <zorder>type_second</zorder>
  </widget>
  <widget class="QGroupBox" name="box_result">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>571</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(124, 191, 229);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;.SF NS Text&quot;;
color: rgb(110, 110, 110);</string>
    </property>
    <property name="text">
     <string>详情展示</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="area_detail">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>551</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Adobe Arabic&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;.SF NS Text&quot;;
border-style:solid;
border-width:2px;
border-color: rgb(124, 191, 229);
border-right-color: rgb(255, 255, 255);
color: rgb(110, 110, 110);</string>
    </property>
    <property name="text">
     <string>编号</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>551</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_brief">
     <item>
      <widget class="QScrollArea" name="area_brief">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>547</width>
          <height>297</height>
         </rect>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>80</y>
           <width>391</width>
           <height>107</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_detail_2">
            <property name="styleSheet">
             <string notr="true">font: 18pt &quot;.SF NS Text&quot;;
color: ;
color: rgb(170, 85, 0);</string>
            </property>
            <property name="text">
             <string>系统配置说明：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_detail_3">
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;.SF NS Text&quot;;
color: ;
color: rgb(170, 85, 0);</string>
            </property>
            <property name="text">
             <string>1）用户可修改大部分标签内容</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_detail_4">
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;.SF NS Text&quot;;
color: ;
color: rgb(170, 85, 0);</string>
            </property>
            <property name="text">
             <string>2）用户可编辑条件搜索选项</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_detail_5">
            <property name="styleSheet">
             <string notr="true">font: 16pt &quot;.SF NS Text&quot;;
color: ;
color: rgb(170, 85, 0);</string>
            </property>
            <property name="text">
             <string>3）保存修改之后，需要重启软件才可生效</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <zorder>label_5</zorder>
        <zorder>func_exit</zorder>
        <zorder>verticalLayoutWidget_2</zorder>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_detail">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(204, 204, 204);
font: 30pt &quot;.SF NS Text&quot;;</string>
    </property>
    <property name="text">
     <string>展示板</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_type_third">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#label_type_third
{
  background-color:rgba(255, 255, 255, 0);
  border-style:none;
  font: 16pt &quot;.SF NS Text&quot;;
}
QLineEdit#label_type_third:focus
{ 
    background-color:rgb(255, 255, 127);
}</string>
    </property>
    <property name="text">
     <string>type3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_type_fourth">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#label_type_fourth
{
background-color: rgba(255, 255, 255, 0);
border-style:solid;
border-width:2px;
border-color: rgba(124, 191, 229, 0);;
border-right-color: rgba(255, 255, 255, 100);
border-left-color: rgba(255, 255, 255, 100);
font: 16pt &quot;.SF NS Text&quot;;
}
QLineEdit#label_type_fourth:focus
{ 
    background-color:rgb(255, 255, 127);
}</string>
    </property>
    <property name="text">
     <string>type4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="box_function">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>191</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>50</y>
      <width>191</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="func_add">
       <property name="font">
        <font>
         <family>.SF NS Text</family>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit#func_add
{
background-color: rgba(255, 255, 255, 0);
font: 18pt &quot;.SF NS Text&quot;;
color: rgb(57, 143, 227);
border-style:none;
}
QLineEdit#func_add:focus
{
	background-color: rgb(255, 255, 127);
}</string>
       </property>
       <property name="text">
        <string>添加  XXXX</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="func_edit">
       <property name="font">
        <font>
         <family>.SF NS Text</family>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit#func_edit
{
background-color: rgba(255, 255, 255, 0);
font: 18pt &quot;.SF NS Text&quot;;
color: rgb(57, 143, 227);
border-style:none;
}
QLineEdit#func_edit:focus
{
	background-color: rgb(255, 255, 127);
}</string>
       </property>
       <property name="text">
        <string>修改  XXXX</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="func_del">
       <property name="font">
        <font>
         <family>.SF NS Text</family>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit#func_del
{
background-color: rgba(255, 255, 255, 0);
font: 18pt &quot;.SF NS Text&quot;;
color: rgb(57, 143, 227);
border-style:none;
}
QLineEdit#func_del:focus
{
	background-color: rgb(255, 255, 127);
}</string>
       </property>
       <property name="text">
        <string>删除  XXXX</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="func_save">
       <property name="font">
        <font>
         <family>.SF NS Text</family>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit#func_save
{
background-color: rgba(255, 255, 255, 0);
font: 18pt &quot;.SF NS Text&quot;;
color: rgb(57, 143, 227);
border-style:none;
}
QLineEdit#func_save:focus
{
	background-color: rgb(255, 255, 127);
}</string>
       </property>
       <property name="text">
        <string>保存  XXXX</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="func_exit">
       <property name="font">
        <font>
         <family>.SF NS Text</family>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 18pt &quot;.SF NS Text&quot;;
color: rgb(110, 110, 110);
</string>
       </property>
       <property name="text">
        <string>退出查询系统</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(150, 160, 20);
font: 20pt &quot;.SF NS Text&quot;;
color: rgb(110, 110, 110);</string>
    </property>
    <property name="text">
     <string>基本功能</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>771</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="title">
      <property name="styleSheet">
       <string notr="true">QLineEdit#title
{
background-color:rgba(255, 255, 255, 0);
border-style:none;
font: 25pt &quot;Adobe Arabic&quot;;
color:#1493f7;
font-weight:bold;
}
QLineEdit#title:focus
{
	background-color: rgb(255, 255, 127);
}</string>
      </property>
      <property name="text">
       <string>Title</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>640</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;.SF NS Text&quot;;
background-color: rgb(240, 252, 252);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>640</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;.SF NS Text&quot;;
background-color: rgb(98, 160, 226);</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
